test/regression/issue/3595-wildcard-side-effects.test.js

```diff
@@ -0,0 +1,533 @@
+import { expect, test } from "bun:test";
```

Comment:
The issue you probably meant to use is #21034 (and I'm sure others), not this random one